script.js

```diff
@@ -0,0 +1,173 @@
+// First-Card
+const title1 = 'Tonic';
+const subtitles1 = ['Canopy', 'Back End Dev', '2015'];
+const description1 = 'A daily selection of privately personalized reads; no accounts or sign-ups required.';
+const imgUrl1 = './images/Snapshoot Portfolio.png';
+const technologies1 = ['html', 'css', 'javascript'];
+const projectData1 = {
+  title: title1,
+  description: description1,
+  image: imgUrl1,
+  technologies: technologies1,
+  subtitles: subtitles1,
+};
```

Comment:
[OPTIONAL] Instead of this, I would advise you to add everything in one array and organize it into objects in the array. With this, you can use for loop to iterate over it.